dev/preview/infrastructure/harvester/cert.tf

```diff
@@ -16,7 +16,7 @@ resource "kubernetes_manifest" "cert" {
       dnsNames = [
         "${var.preview_name}.preview.gitpod-dev.com",
         "*.${var.preview_name}.preview.gitpod-dev.com",
-        "*.ws.${var.preview_name}.preview.gitpod-dev.com"
+        "*.ws-dev.${var.preview_name}.preview.gitpod-dev.com"
```

Comment:
Let's keep both of them for a bit, to make sure nothing has been missed and they are not required anymore. That will provide a smoother migration for existing envs as well.

Reply:
fixed